docs/changelog.txt

```diff
@@ -59,6 +59,7 @@ Template for new versions:
 ## Fixes
 
 ## Misc Improvements
+- Translate: will use DF's ``translate_name`` function, if available, instead of the DFHack emulation
```

Comment:
since this should be transparent to the end user, I don't think we need a changelog for this, at least not in this section. If it goes anywhere, it should be in an "Internals" section

Reply:
going to move it under API